tms_objects2.erb

```diff
@@ -432,6 +439,8 @@ objects_main_query = tmsclient.execute("SELECT * FROM [Coboat_ObjectsMainQuery]
 #7 - eventMaterialsTech - conceptID/term - Has all(colThesxrefType,UCTS,FinalST,term) 
 #6113 - ExhTextEntries |t| - eventID - TextType - LUX exhibition
 #4999 - ExhTextEntries |t| - eventID - TextType - ExhibitionURL
+#586 - ExhVenuesAltName |an| - nameActorSet - appellationValue - AltVenueName - NULL
+#ExhVenuesAltName |an| - nameActorSet - appellationValue - NameType - (No NULL -> NameType)
```

Comment:
I think clearest was to describe - remove 442, and 443 and put:

# if statement for ExhVenuesAltName |an| - nameActorSet - appellationValue - always occurs

Reply:
never mind keep 442 and 443